.github/workflows/main.yml

```diff
@@ -85,7 +85,7 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          pip install pixl_core/[test]
+          pip install pytest-pixl pixl_core/[test]
```

Comment:
I wonder if this runs in order or dependencies, if so then it'll need to install core first